lib/rubocop/cop/style/method_call_without_args_parentheses.rb

```diff
@@ -44,6 +44,7 @@ def on_send(node)
 
           register_offense(node)
         end
+        alias on_csend on_send
         # rubocop:enable Metrics/CyclomaticComplexity
```

Comment:
That's my two cents. The offending range might be covered with the following. :-)

```suggestion
        # rubocop:enable Metrics/CyclomaticComplexity
        alias on_csend on_send
```